src/kvstore/NebulaStore.cpp

```diff
@@ -1133,7 +1133,7 @@ void NebulaStore::cleanWAL() {
       auto& part = partEntry.second;
       if (part->needToCleanWal()) {
         // clean wal by expired time
-        part->wal()->cleanWAL();
+        part->cleanWal();
```

Comment:
Maybe unify all usage, either implement `cleanWAL()` in `Part` and `Listener`, or just call `part->wal()->cleanWAL(id)`.